data/assets.yaml

```diff
@@ -85,54 +85,90 @@ shapefiles:
     prj: 4326
     tile: true
 
+  # TEMPORARY DISABLE TO PULL IN v5-PRE4 DATA
+  #
+  # - name: ne_10m_populated_places
+  #   url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_populated_places.zip
+  #   prj: 4326
+  #
+  # - name: ne_110m_admin_0_boundary_lines_land
+  #   url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/110m/cultural/ne_110m_admin_0_boundary_lines_land.zip
+  #   prj: 4326
+  #
+  # - name: ne_50m_admin_0_boundary_lines_land
+  #   url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/50m/cultural/ne_50m_admin_0_boundary_lines_land.zip
+  #   prj: 4326
+  #
+  #- name: ne_10m_admin_0_boundary_lines_land
+  #  url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_admin_0_boundary_lines_land.zip
+  #  prj: 4326
+  #
+  #- name: ne_10m_admin_0_boundary_lines_map_units
+  #  url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_admin_0_boundary_lines_map_units.zip
+  #  prj: 4326
+  #
+  # - name: ne_50m_admin_1_states_provinces_lines
+  #   url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/50m/cultural/ne_50m_admin_1_states_provinces_lines.zip
+  #   prj: 4326
+  #   shapefile-name: ne_50m_admin_1_states_provinces_lines.shp
+
+  # - name: ne_10m_admin_1_states_provinces_lines
+  #   url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_admin_1_states_provinces_lines.zip
+  #   prj: 4326
+
   - name: ne_10m_populated_places
-    url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_populated_places.zip
```

Comment:
Should we keep these source URLs but commented out (like above) so it's easy to add them back in once NE v5 is released?

Reply:
Yup, I kept the [whole block, but commented out](https://github.com/tilezen/vector-datasource/pull/1896/files#diff-211ab6b26f210fbe082be63a883f0a0cR91). That way it's easy to switch out the whole thing and comment out the new stuff, without having to comment / uncomment a bunch of lines individually.

Reply:
Ah, Github.com diff display not winning here. I see it now, tnx.

Reply:
Yea, sorry. I guess that's the advantage of changing each line individually; the diff is much easier to read. :bowing_man: